RibonDAO/manager
# Tech 137 Create and Edit Cause

# Description

- Add and edit causes
- Create a Pool when a cause is created, so that is why we have a confirmation modal on creation

## Type of change

Please delete options that are not relevant.


## How to test

Put here all the things need to test this PR and verify your changes, also list any relevant details for tests (eg: have a wallet)

- Try to create a cause, but remember that it creates a pool on our wallet
- Try to edit a cause, it doesn't change the pool
